src/pyOpenMS/pxds/IDFilter.pxd

```diff
@@ -19,82 +19,146 @@ cdef extern from "<OpenMS/FILTERING/ID/IDFilter.h>" namespace "OpenMS":
-        void filterHitsByScore(libcpp_vector[ProteinIdentification]& ids, double threshold_score) nogil except +
+        void filterHitsByScore(libcpp_vector[PeptideIdentification]& ids, double threshold_score) nogil except + # wrap-doc:Filters peptide or protein identifications according to the score of the hits. Only peptide/protein hits with a score at least as good as 'threshold_score' are kept. Score orientation (are higher scores better?) is taken into account
+        void filterHitsByScore(libcpp_vector[ProteinIdentification]& ids, double threshold_score) nogil except + # wrap-doc:Filters peptide or protein identifications according to the score of the hits. Only peptide/protein hits with a score at least as good as 'threshold_score' are kept. Score orientation (are higher scores better?) is taken into account
 
-        void keepNBestSpectra(libcpp_vector[PeptideIdentification]& peptides, Size n) nogil except +
+        void keepNBestSpectra(libcpp_vector[PeptideIdentification]& peptides, Size n) nogil except + # wrap-doc:Filter identifications by "N best" PeptideIdentification objects (better PeptideIdentification means better [best] PeptideHit than other)
 
-        void keepNBestHits(libcpp_vector[PeptideIdentification]& ids, Size n) nogil except +
-        void keepNBestHits(libcpp_vector[ProteinIdentification]& ids, Size n) nogil except +
+        void keepNBestHits(libcpp_vector[PeptideIdentification]& ids, Size n) nogil except + # TODO
+        void keepNBestHits(libcpp_vector[ProteinIdentification]& ids, Size n) nogil except + # TODO
 
         void filterHitsByRank(libcpp_vector[PeptideIdentification]& ids, Size min_rank, Size max_rank) nogil except +
+            # wrap-doc:
+                #   Filters peptide or protein identifications according to the ranking of the hits
+                #   -----
+                #   The hits between 'min_rank' and 'max_rank' (both inclusive) in each ID are kept.
+                #   Counting starts at 1, i.e. the best (highest/lowest scoring) hit has rank 1.
+                #   The ranks are (re-)computed before filtering.
+                #   'max_rank' is ignored if it is smaller than 'min_rank'.
+                #   -----
+                #   Note: that there may be several hits with the same rank in a peptide or protein ID (if the scores are the same).
+                #   This method is useful if a range of higher hits is needed for decoy fairness analysis.
+
         void filterHitsByRank(libcpp_vector[ProteinIdentification]& ids, Size min_rank, Size max_rank) nogil except +
+            # wrap-doc:
+                #   Filters peptide or protein identifications according to the ranking of the hits
+                #   -----
+                #   The hits between 'min_rank' and 'max_rank' (both inclusive) in each ID are kept.
+                #   Counting starts at 1, i.e. the best (highest/lowest scoring) hit has rank 1.
```

Comment:
rm dot

Reply:
Multiline statement. Can we ignore it?

Reply:
Yes, but then this has to be consistent.